cmd/worker/worker.go

```diff
@@ -53,14 +53,13 @@ func NewWorkerCmd() *cobra.Command {
 	or CLI flag:
 	$ k0s worker --token-file [path_to_file]
 	Note: Token can be passed either as a CLI argument or as a flag`,
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			logrus.SetOutput(cmd.OutOrStdout())
+			logrus.SetLevel(logrus.InfoLevel)
```

Comment:
If root's prerun does this, I don't think the subcommands need to do it on their own

Reply:
Bummer that I've forgotten that I've set it to auto-merge :see_no_evil: Will fix this as well.

Reply:
Ah, no. I remember why it's there: Those three subcommands have info as their default level, whereas all others have warn as default. Setting the level before the call to the parent persistent prerun saves some extra logic to enable debug logging when requested.